src/project-tests/bar-chart-tests.js

```diff
@@ -1,196 +1,274 @@
+  getShapeValueYearBar,
+  getShapeValueDecimal,
+  getTickPosition,
+  getTickValueThousands,
+  getTickValueYear
+} from '../utils/alignment-D3-support';
+
 import { assert } from 'chai';
 import { testToolTip } from '../utils/global-D3-tests';
 import $ from 'jquery';
+import { d3ProjectStack } from '../utils/shared-test-strings';
 
 export default function createBarChartTests() {
 
   describe('#BarChartTests', function() {
-    let reqNum = 0;
-
-    reqNum++;
-    it(`${reqNum}. My chart should have a title with a corresponding
-    id="title"`,
-    function() {
-      assert.isNotNull(
-        document.getElementById('title'),
-        'Could not find element with id="title" '
-      );
-    });
 
-    reqNum++;
-    it(`${reqNum}. My Chart should have an x-axis with a corresponding
-    id="x-axis"`,
-    function() {
-      assert.isNotNull(
-        document.getElementById('x-axis'),
-        'Could not find element with id="x-axis" '
-      );
-      assert.isAbove(
-        document.querySelectorAll('g#x-axis').length,
-        0,
-        'x-axis should be a <g> SVG element '
-      );
+    describe('#Technology Stack', function() {
+      it(d3ProjectStack, function() {
+        return true;
+      });
     });
 
-    reqNum++;
-    it(`${reqNum}. My Chart should have a y-axis with a corresponding
-    id="y-axis"`,
-    function() {
-      assert.isNotNull(
-        document.getElementById('y-axis'),
-        'Could not find element with id="y-axis" '
-      );
-
-      assert.isAbove(
-        document.querySelectorAll('g#y-axis').length,
-        0,
-        'y-axis should be a <g> SVG element '
-      );
-    });
+    describe('#Content', function() {
+      let reqNum = 0;
 
-    reqNum++;
-    it(`${reqNum}. Both axes should contain multiple tick labels`,
-    function() {
-      assert.isAbove(
-        $('#x-axis .tick').length,
-        1,
-        'There are not enough tick labels on the x-axis '
-      );
-      assert.isAbove(
-        $('#y-axis .tick').length,
-        1,
-        'There are not enough tick labels on the y-axis '
-      );
-    });
+      reqNum++;
+      it(`${reqNum}. My chart should have a title with a corresponding
+      id="title"`,
+      function() {
+        assert.isNotNull(
+          document.getElementById('title'),
+          'Could not find element with id="title" '
+        );
+      });
 
-    reqNum++;
-    it(`${reqNum}. My Chart should have a bar for each data point with a
-    corresponding class="bar" displaying the data`,
-    function() {
-      assert.isAbove(
-        document.querySelectorAll('rect.bar').length,
-        0,
-        'Could not find any elements with class="bar" '
-      );
-      assert.equal(
-        document.querySelectorAll('rect.bar').length,
-        275,
-        'The number of bars is not equal to the number of data points '
-      );
-    });
+      reqNum++;
+      it(`${reqNum}. My Chart should have element x-axis with a corresponding
```

Comment:
This description is still missing that the Camper needs to create a <g> element. It's a very small change, but it makes it clearer what we are looking for:

> My Chart should have a \<g\> element x-axis with a corresponding